agent/Makefile

```diff
@@ -255,6 +255,7 @@ install-python3-setup: install-util-scripts install-lib
 	${COPY} ${DESTDIR}/python3/bin/pbench-clear-results ${UTILDIR}/
 	${COPY} ${DESTDIR}/python3/bin/pbench-cleanup ${UTILDIR}/
 	${COPY} ${DESTDIR}/python3/bin/pbench-list-tools ${UTILDIR}/
+	${COPY} ${DESTDIR}/python3/bin/pbench-register-tool-triggers ${UTILDIR}/
```

Comment:
Don't we also have to remote it from [line 49](https://github.com/distributed-system-analysis/pbench/pull/1883/files#diff-34cc66de7395e97215a24040803ff3e7ab5dd07835a701dbb6a858012f696f7eL49)?

Reply:
This still needs to be addressed.